prometheus-ksonnet/grafana/grr.libsonnet

```diff
@@ -0,0 +1,48 @@
+/*
+ * The `grr` function below will render Grafana resources generated by this
+ * Grafana library (e.g. as `grafanaDashboards`) into Kubernetes style objects,
+ * with kind/apiVersion. (Grizzly)[https://github.com/grafana/grizzly)
+ * has the ability to consume resources in this format.
```

Comment:
I like this, can you make it a separate library? The concept of $.mixins is bigger than prometheus-ksonnet.

Reply:
This is fair enough, and your suggestion below about a library api works around my objections.